keyboards/keyquest/enclave/config.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x1117
+#define PRODUCT_ID      0x0E0E
+#define DEVICE_VER      0x0001
+#define PRODUCT         Enclave-1
+
+/* key matrix size */
+#define MATRIX_ROWS 3
+#define MATRIX_COLS 3
+
+/* Keyboard Matrix Assignments*/
+#define MATRIX_ROW_PINS { D6, B6, F5 }
+#define MATRIX_COL_PINS { B4, B7, C7 }
+#define UNUSED_PINS
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION COL2ROW
+
+/* RGB light setup*/
+#define RGB_DI_PIN B5
+#define RGBLED_NUM 2
+#define RGBLIGHT_ANIMATIONS
```

Comment:
Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```